server/base/src/test/java/org/apache/accumulo/server/conf/util/ZooInfoViewerTest.java

```diff
@@ -148,14 +154,23 @@ public void instanceIdOutputTest() throws Exception {
 
     String testFileName = "./target/zoo-info-viewer-" + System.currentTimeMillis() + ".txt";
 
-    ZooInfoViewer.Opts opts = new ZooInfoViewer.Opts();
-    opts.parseArgs(ZooInfoViewer.class.getName(),
-        new String[] {"--print-instances", "--outfile", testFileName});
+    ServerContext context =
+        MockServerContext.getWithZK(InstanceId.of(instanceName), instanceName, 20_000);
+    expect(context.getZooReader()).andReturn(zooReader).once();
```

Comment:
if you add `context.close()` on line 160 this might fix the issue. It should register that the method should be called on the Mock object.